src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,7 +1,95 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import DetailSize from '../../components/DetailSize/DetailSize';
+import './ProductDetail.scss';
 
 const ProductDetail = () => {
-  return <div></div>;
+  const [data, setData] = useState({});
+
+  const params = useParams();
+
+  useEffect(() => {
+    fetch('./data/detail.json')
+      // fetch(`http://10.58.7.243:8000/products/${params.id}`)
+      .then(response => response.json())
+      .then(data => setData(data.result));
+  }, []);
+  // }, [params.id]);
+
+  const addCommaToPrice = price => {
+    return price.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',');
+  };
+  const [minPrice, maxPrice] = [
+    data.description && addCommaToPrice(data.options[0].price),
+    data.description &&
+      addCommaToPrice(data.options[data.options.length - 1].price),
+  ];
```

Comment:
```suggestion
  const [minPrice, maxPrice] = [
    Number(data.id && data?.options.at(0)?.price).toLocaleString(),
    Number(data.id && data?.options.at(-1)?.price).toLocaleString(),
  ];
```
  숫자 3개마다 점찍는걸 구현하시려고 한다면 이 방법이 더 깔끔해보입니다
toLocaleString() 메서드를 사용하면 위의 함수는 제거해도 되겠죠?

Reply:
네 감사합니다. 적용하였습니다 ~